openquake/hazardlib/sourceconverter.py

```diff
@@ -749,6 +749,17 @@ def get_tom(self, node):
         :returns: a :class:`openquake.hazardlib.mfd.EvenlyDiscretizedMFD.` or
                   :class:`openquake.hazardlib.mfd.TruncatedGRMFD` instance
         """
+
+        if node.tag.endswith('pointSource'):        # Check if there a tom specified at the point level
```

Comment:
This is not clear to me: what happens if you have a non-pointsource? How do you specify the TOM? Also, if you have 1 million sources specifying the TOM for each one makes little sense. My understanding is the TOM should be set at the SourceGroup level, not at the source level, i.e. there are multiple sources with the same TOM. Let me check with @mmpagani and we will implement that feature, including a general way of passing extra parameters to the TOM subclass.

Reply:
It turns out it is already implemented, see this example: https://github.com/gem/oq-engine/blob/master/openquake/qa_tests_data/classical/case_35/source_model.xml
The place to touch is the method `get_tom` in the SourceConverter.